Consumer.py

```diff
@@ -0,0 +1,56 @@
+from kafka import KafkaConsumer
+from pymongo import MongoClient
+from json import loads
+from flask_pymongo import PyMongo
+import json
+from datetime import datetime
+import logging
+import sys
+from json import dumps
+from time import sleep
+from pymongo import MongoClient
+import random
+from flask import Flask, flash, render_template, request, jsonify
+from flask_cors import CORS, cross_origin
+from flask_pymongo import PyMongo
+
+
+app= Flask(__name__)
+app.config['MONGO_DBNAME'] = 'offtop-kafka-mongodb'
+app.config['MONGO_URI'] = 'mongodb+srv://off-top:offtoppassword@off-top-kafka-mogsf.mongodb.net/off-top'
+app.config['MONGO_USER'] = 'off-top'
+app.config['MONGO_PASSWORD'] = 'offtoppassword'
+mongo = PyMongo()
+mongo.init_app(app)
+
+def startConsumer():
+    consumer= KafkaConsumer(
+        'OutgoingFocusAlert',
```

Comment:
This consumer is supposed to be receiving data from another microservice, currently, it would be from Producer.py in off-top-python, so rename this topic to something like 'IncomingFocusAlert`

Reply:
This also means that in Producer.py of off-top-python you will need to check if the focus_score is false to send that data to the topic of `NewFocusAlert`